app/src/main/java/com/orange/ods/app/ui/components/bottomnavigation/ComponentBottomNavigation.kt

```diff
@@ -91,24 +88,28 @@ fun ComponentBottomNavigation() {
                         }
                     )
                 }
-            }
+            )
 
             CodeImplementationColumn(
                 modifier = Modifier.padding(horizontal = dimensionResource(id = R.dimen.screen_horizontal_margin))
             ) {
-                FunctionCallCode(name = OdsComposable.OdsBottomNavigation.name) {
-                    navigationItems.take(2).forEach { item ->
-                        FunctionCallCode(
-                            name = OdsComposable.OdsBottomNavigationItem.name,
-                            parameters = {
-                                icon()
-                                string("label", context.getString(item.textResId))
-                                selected(selectedNavigationItem.value.textResId == item.textResId)
-                                onClick()
+                FunctionCallCode(
+                    name = OdsComposable.OdsBottomNavigation.name,
+                    parameters = {
+                        list("items") {
+                            navigationItems.take(2).forEach { item ->
```

Comment:
I dont know if it's normal we only take the 2 first items? When increasing navigation item count in the customization bottom sheet, nothing changes in code implementation section.